chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+This \textbf{master dataset}\sidenote{\url{
+		https://dimewiki.worldbank.org/Master\_Data\_Set}}
+should contain a record of all units ever encountered for each relevant level of observation,
+and their corresponding unique identifiers.
+
+
+% Merchand
+Checking for duplicated entries and making sure that each observation has a unique identifier are usually part of data quality monitoring.
+The lack of a unique identifier should be addressed as soon as data is received,
+so you can keep track of observations.
+\texttt{ieduplicates} and \texttt{iecompdup},
+two Stata commands included in the \texttt{iefieldkit}
+package\index{iefieldkit},\sidenote{\url{https://dimewiki.worldbank.org/iefieldkit}}
+create an automated workflow to identify, correct and document
+occurrences of duplicate entries.
+One advantage of using \texttt{ieduplicates}\sidenote{\url{
+		https://dimewiki.worldbank.org/wiki/ieduplicates}} 
+to correct duplicated entries is that it creates \textit{duplicates reports}
+that record which corrections made and why.
+Even if you are not using this command,
+it is important to include similar records in the \texttt{Documentation} folder.
+
+\subsection{What makes data ``tidy''}
+
+% Where these concepts come from
+Before discussing the details of how to clean your data,
+we need to define a few concepts that will be used in this chapter.
+They refer to the structure of the data,
+and have long been discussed in the database management literature.
+This section is based on Hadley Wickham's definition of tidy data,\cite{tidy-data}
+which uses a terminology that is more intuitive for users of applied statistics,
+we have added our own twist based on the jargon of development research.
+
+% Definitions
+Development data is typically acquired in the form of \textbf{tables},
+two-dimensional datasets formed by rows and columns.\sidenote{\textbf{Table:}
+	a two-dimensional dataset formed by rows and columns. 
+	Also called ``rectangular data''.
+	One example of non-rectangular data is a text.}
+Each element or cell in this table is a \textbf{data point}, that is,
+a value representing a measure of an attribute for one individual unit.\sidenote{\textbf{Data point:}
+a value representing a measure of an attribute for one individual unit.}
+For example, in a household survey,
+two of the attributes usually measured are gender and age.
+The measures of gender and age for each household member are data points.
+A \textbf{variable}\sidenote{\textbf{Variable:} the collection of all data points that measure the same attribute.}
+ is the collection of all data points that measure a given attribute.
+In the example of a household survey,
+each variable will correspond to one survey question.
+Finally, each variable will contain data points for different individuals in a given unit of observation.
+In our example, each household member is a subject, and the unit of observation is a person.
```

Comment:
Unit of observation in a household survey is more often a household than a person.

Reply:
Right, it was meant to be a household roster. fixed in 3ee3f81